.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["2.7", "3.6", "3.7"]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install dependencies
+        run: pip install -r requirements-test.txt
+      - name: Install RPM tools
+        run : |
+          curl --create-dirs -o ~/.local/bin/spectool https://pagure.io/rpmdevtools/raw/26a8abc746fba9c0b32eb899b96c92841a37855a/f/spectool.in
```

Comment:
Would it be easier to use a Fedora container to execute in (since actions allows that)?

Reply:
I'd use CentOS, not Fedora, but yes, I thought about that. Probably better.

Reply:
but of course, if I now try to use pip, it fails as the right python version isn't there anymore, whaaaa.

for another day.

Reply:
I'd like to leave that for another day.

Reply:
Yes, I think I also ran into the problem that when using a specific container, things like language runtimes (Python, Ruby, probably all others) no longer work. It probably makes sense if they just deploy a tarball that things would break due to linking.